src/foam/dao/history/view/HistoryRecordView.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.history.view',
+  name: 'HistoryRecordView',
+  extends: 'foam.u2.View',
+
+  documentation: `
+    A view for displaying history records for specifed object(record) id and specificied property.
+    daoKey, objectId, and propertyName must be provided when creating this view
+
+    e.g., if you want to display compliance history from user's history records for user with id 1111,
```

Comment:
Here and below, we reference Compliance.  This is the generic view.  
Perhaps the view can take a title or name property, assuming it's used in the display.

Reply:
The view does take property name to make it generic